### PR TITLE
attempt to make the doc folder less overwhelming by moving platform s…

### PR DESCRIPTION
**Pull request by rdp**
…pecific docs to their own folder
#386
thanks!

### CODE REVIEW

File: Makefile.am [hunk 1]
```diff
@@ -214,35 +214,35 @@ dist_doc_DATA = \
     AUTHORS \
     LICENSE \
     README.md \
-    doc/README.DGUX386 \
-    doc/README.Mac \
-    doc/README.OS2 \
-    doc/README.aix \
-    doc/README.amiga \
-    doc/README.arm.cross \
+    doc/platforms/README.DGUX386 \
```
Comment (ivmai) [inline Makefile.am:217]: Should we also update cmake script?

File: Makefile.am [hunk 2]
```diff
@@ -214,35 +214,35 @@ dist_doc_DATA = \
...
+    doc/platforms/README.ews4800 \
+    doc/platforms/README.hp \
+    doc/platforms/README.linux \
+    doc/platforms/README.sgi \
+    doc/platforms/README.solaris2 \
+    doc/platforms/README.symbian \
+    doc/platforms/README.uts \
+    doc/platforms/README.win32 \
+    doc/platforms/README.win64 \
+    doc/platforms/porting.md \
```
Comment (ivmai) [inline Makefile.am:234]: Let's don't move porting.md

File: doc/simple_example.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ a basic correctness test which usually takes well under a minute.
 ### Other platforms
 
 On non-Unix, non-Linux platforms, the collector is usually built by copying
-the appropriate makefile (see the platform-specific README in doc/README.xxx
+the appropriate makefile (see the platform-specific README in doc/platforms/README.xxx
```
Comment (ivmai) [inline doc/simple_example.md:34]: doc/platforms/README.xxx -> doc/platforms

File: README.md [hunk 4]
```diff
@@ -144,7 +144,7 @@ areas that are associated with dynamic libraries.  This is easy to remedy
 if you know how to find those data areas on your operating system (see
 `GC_add_roots`).  Code for doing this under SunOS, IRIX 5.X and 6.X, HP/UX,
 Alpha OSF/1, Linux, and Win32 is included and used by default.
-(See [README.win32](doc/README.win32) and [README.win64](doc/README.win64)
+(See [README.win32](doc/platforms/README.win32) and [README.win64](doc/platforms/README.win64)
```
Comment (ivmai) [inline README.md:147]: Please keep lines <79 chars long.

File: Makefile.am (hunk 1 above)
Comment (rdp) [reply Makefile.am:217]: It looks as if it uses a glob so it seemed to still work for me...

File: doc/simple_example.md (hunk 3 above)
Comment (rdp) [reply doc/simple_example.md]: OK

File: README.md (hunk 4 above)
Comment (rdp) [reply README.md]: OK

**Comment by ivmai** [comment]
Thank you. Squashed and committed (with small modification) as commit d98402573

**Pull request closed without merging**